### ISSUE TITLE
Plan a11y validation testing

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#1152]
- [x] Draft the [Research plan](https://docs.google.com/document/d/19EZLdDU-_4fGym1q-_MVi1ygT16amJ4HIzYKNUJDUi4/edit#)
- [x] Draft the [facilitation guide](https://docs.google.com/document/d/1wDKGIvYvES6e1JcYNCd8nVzjSESJDDAqQNMBEzja530/edit#)
- [x] Review the facilitation guide with devs from a security perspective
- [x] Send facilitation guide to translation
- [x] Draft the [privacy statement](https://docs.google.com/document/d/11UzfqMKfBG7j5zGQGfHyPVTkwOn0MevEOxlB3U98-ak/edit#)
- [x] Privacy statement is reviewed from policy perspective
- [x] Send privacy statement to translation
- [x] Publish privacy statements to GC Articles
- [x] Draft the [request description](https://docs.google.com/document/d/1lYCbx7LzaT08F25aPUfTaBuONUoWFHazCeFcvsTMqpo/edit)
- [x] Send request description to translation
- [x] Create secret link to include in request
- [x] Post the invite on Fable
- [x] Create note taker templates
- [x] Assign note takers for sessions
- [x] Prep note takers and walk through note taking template

**Comment by andrewleith** [comment]
- Work ongoing; items sent to translation
- Security considerations discussed with Will

**Comment by YedidaZalik** [comment]
Adrianne on vacation, Yael will check with Abdul late in the week for unused slots and Phil has some other tasks.

**Comment by amazingphilippe** [comment]
All 10 Fable requests submitted

8 requests scheduled.

2 French requests to be scheduled.

**Comment by YedidaZalik** [comment]
Phil has added the above to Adrianne's calendar. 
Need to determine whether to invite note-takers, and they're working on getting French participants.
If anyone particularly interested in note-taking reach out to Phil, it depends on whether we can include others through Fable. 
Phil will post dates and we can react based on dates. We have zoom link so can invite anyone.
Recommended to all to try this.

**Comment by YedidaZalik** [comment]
All 10 participants booked and Phil will post in slack to team to join.

**Comment by YedidaZalik** [comment]
Phil says enough note-takers and observers and there's still room for folks to join if they'd like
